src/Form.js

```diff
@@ -43,15 +41,15 @@ class Form extends Component {
     const {
       className,
       cssModule,
-      inline,
       tag: Tag,
       innerRef,
       ...attributes
     } = this.props;
 
-    const classes = mapToCssModules(
-      classNames(className, inline ? 'form-inline' : false),
-      cssModule,
+    const classes = mapToCssModules(className, cssModule);
+
+    return (
+      <Tag {...attributes} ref={innerRef} className={classes} />
     );
 
```

Comment:
Multiple return statements doing the same thing.

Reply:
could have sworn I resolved this yesterday... I'll get it fixed. Sorry!